packages/core/upload/server/content-types/folder/schema.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+module.exports = {
+  collectionName: 'folders',
+  info: {
+    singularName: 'folder',
+    pluralName: 'folders',
+    displayName: 'Folder',
+  },
+  options: {},
+  pluginOptions: {
+    'content-manager': {
+      visible: false,
+    },
+    'content-type-builder': {
+      visible: false,
+    },
+  },
+  attributes: {
+    name: {
+      type: 'string',
+      min: 1,
+      required: true,
+    },
+    uid: {
+      type: 'string',
```

Comment:
Do we have something in place already to use native UUID database features, that we could use instead?

Reply:
Not from my knowledge. We have the `uid` type but it is just a custom type that is unique and can be based on another field (`title: "My friend is a kangaroo", uid: "my-friend-is-a-kangaroo"`).
Do you have something in mind?

Reply:
I was thinking if we could move the uuid generation into the database level instead of the application, but apparently neither `MySQL` nor `SQlite` support it out of the box, so this is fine.